hawc/apps/lit/models.py

```diff
@@ -868,6 +868,15 @@ class Reference(models.Model):
 
     BREADCRUMB_PARENT = "assessment"
 
+    def update_tags(self, tag_pks, user):
+        user_ref_tags, created = UserReferenceTag.objects.get_or_create(reference=self, user=user)
+        user_ref_tags.tags.set(tag_pks)
+        user_ref_tags.save()
+        if not self.assessment.literature_settings.conflict_resolution:
+            self.tags.set(tag_pks)
+        else:
+            raise NotImplementedError
```

Comment:
I know it wasn't on the JIRA card, but do you think you could take a crack at writing this? 

- Query the UserReferenceTag and get all reviews for this reference. 
- If <2 reviews exist, return; no change
- Get a set of tag ids from all reviews; if all sets of tag ids are not the same; no change
- If all tags are the same, save tags on reference and update last_updated

That's probably what we need for now. I'm thinking ahead though, and when we add a view to find all conflicts, this step would be really expensive. Therefore, I wonder if we may end up adding another field to UserReferenceTag for `resolved` might be good, where basically it's set to False until this operation is complete...